examples/stargate-jwt-auth/docker-compose.yml

```diff
@@ -45,11 +45,18 @@ services:
     networks:
       - backend
     ports:
+      # cql
+      - 9045:9042
+      # graphql
       - 8080:8080
+      # auth
       - 8081:8081
+      # REST
       - 8082:8082
+      # health
       - 8084:8084
-      - 9049:9042
```

Comment:
This one is slightly inconsistent. Intentional?

Reply:
No, pretty sure that was a mistake.. :+1:

Reply:
Hmm, but I am seeing that the `cass-1` was also exposed on `9048`, so I guess it was intentional..

Reply:
This line maps external port 9049 to internal 9042. cassandra exposes a bunch of ports for internal node-to-node communication, but not all of them need external maps. 9042 is the default cql port, which folks might actually just want to map as-is in most cases (9042:9042) for better compatibility.